dgv/metrics/task_functions.py

```diff
@@ -222,233 +339,186 @@ def get_id(arr, list_obj):
 
 
 def append_chunk(cpt, obj_type, arr, list_obj, log):
-    print(f"{obj_type} : {cpt}")
+    logging.info(f"{obj_type} : {cpt}")
```

Comment:
I think it will be the first time we use logging instead of print in the pipelines 😅 I have in my to-do to remove all prints for logs

Reply:
Niice! 😄